docs/api.md

```diff
@@ -2304,6 +2305,27 @@ const selector = '.foo';
 await page.waitForFunction(selector => !!document.querySelector(selector), {}, selector);
 ```
 
+#### frame.waitForNavigation(options)
```

Comment:
Can you add a message to page.waitForNavigation saying that it’s a shortcut to this method?

Reply:
Done: https://github.com/GoogleChrome/puppeteer/blob/89469192c71d8edbe7dba9cc76ff1907f98f8c3c/docs/api.md#pagewaitfornavigationoptions